.gitignore

```diff
@@ -1,5 +1,9 @@
 # Dependency directory
-node_modules
+/node_modules
 
 # Environment variables
-.env
\ No newline at end of file
+.env
+
+/.idea
+
+.DS_Store
```

Comment:
Your editor doesn't end its files with a newline, which is what that little red glyph at the end is complaining about.

Can you take a look in the settings to make it end with a newline? And if it doesn't delete trailing whitespace already that'd be good.